templates/logged_in.html

```diff
@@ -33,7 +33,7 @@
 			<a href="#">
 			<div class="lewa stats">
 				<p>wolne reputy</p>
-				<p class="freevalue">{{ user.total_cash_formatted }}</p>
+				<p class="freevalue">{{ user.unused_reput }}</p>
```

Comment:
Nie jestem do tego przekonany. Moim zdaniem unused_reput to to samo co total_cash, a tam to jeszcze było formatowane przez metodę